cli/azd/internal/appdetect/appdetect.go

```diff
@@ -156,10 +156,22 @@ func (a AzureDepServiceBus) ResourceDisplay() string {
 
 type AzureDepEventHubs struct {
 	EventHubsNamePropertyMap map[string]string
+	FromDependency           []DependencyType
```

Comment:
For this property naming, Xiaolu has some concern that `appdetect` is language independent, so here we should NOT expose something language specific, like `Maven`. For example, maybe Python language also uses the `FromDependency`, but it should not use `MavenDependencyType`, just enough to use `DependencyType`.

Reply:
How about renaming variable name from `FromDependency` to `DependencyTypes`?

Reply:
Updated.